docs/jobs.rst

```diff
@@ -83,6 +83,11 @@ Optional Fields
     None, which means it will use the default time_zone set in the master
     config.
 
+**expected_runtime** (default **None**)
+    A time interval (ex: "2 hours") that specified the maximal duration of each job run.
```

Comment:
say "maximum expected duration"

Then clarify that this is not a hard limit, and if they need a hard limit, use max_runtime